packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -562,7 +570,7 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
         this.conflictResolver = conflictResolver;
         this.endConflictResolver =
             (key: TInterval, currentKey: TInterval) => {
-                const ival = this.conflictResolver(key, currentKey);
+                const ival = this.conflictResolver?.(key, currentKey);
```

Comment:
somewhat pre-existing, but seems like you could either:

1. Move this assignment to `this.endConflictResolver` to the constructor
2. Reference `conflictResolver` in the closure and get rid of the conditional check.
I'd kind of prefer (2)--it makes it clearer under expected usage this should be defined. Also it's not like usage repeatedly calls this.